tests/phpunit/tests/test-search-form.php

```diff
@@ -70,9 +70,19 @@ public function test_search_block() {
 		global $wp_rewrite;
 
 		$this->frontend->curlang = self::$model->get_language( 'fr' );
-		$form = do_blocks( '<!-- wp:search /-->' );
+		$form                = do_blocks( '<!-- wp:search /-->' );
+		$form_button_outside = do_blocks( '<!-- wp:search {"showLabel":false,"buttonUseIcon":true,"align":"left"} /-->' );
+		$form_button_inside  = do_blocks( '<!-- wp:search {"showLabel":false,"buttonPosition":"button-inside","buttonUseIcon":true,"align":"left"} /-->' );
+		$form_no_button      = do_blocks( '<!-- wp:search {"showLabel":false,"buttonPosition":"no-button","buttonUseIcon":true,"align":"left"} /-->' );
+		$form_button_only    = do_blocks( '<!-- wp:search {"showLabel":false,"buttonPosition":"button-only","buttonUseIcon":true,"isSearchFieldHidden":true,"align":"left"} /-->' );
+
+		$form = do_blocks( '<!-- wp:search {"showLabel":false,"buttonPosition":"button-only","buttonUseIcon":true,"isSearchFieldHidden":true} /-->' );
```

Comment:
Overwriting `$form`?

Reply:
😮 Oops! copy/paste forgotten

Reply:
Done in 64244df1412f71a0a1e3d56bd582207b5da0f61f